mocha-client-tests/addon/manifest.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Mocha tests",
+  "version": "1.0",
+  "manifest_version": 2,
+  "description": "Check ",
+  "icons": {
+    "16": "images/icon-16.png"
+  },
+  "short_name": "MochaTest",
+  "background": {
+    "scripts": [
+      "background.js"
+    ]
+  },
+  "permissions": [
```

Comment:
This looks like it has a bunch of permissions not needed for this example. Can you strip it down to only the relevant permissions? I think only `notifications` is needed, right?